### PR TITLE
add host network

### PR DESCRIPTION
**Pull request by husky-parul**
Signed-off-by: Parul Singh <parsingh@redhat.com>

**Which issue(s) this PR addresses**:
Adds `network=host` to start up podman systemd file

### CODE REVIEW

File: packaging/systemd/microshift-containerized.service [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# microshift
+# container-microshift.service
 # autogenerated by Podman 3.4.0
-# Thu Oct 28 14:18:51 UTC 2021
+# Fri Nov  5 14:30:39 UTC 2021
 
 [Unit]
-Description=MicroShift Containerized
+Description=Podman container-microshift.service
```
Comment (mangelajo) [inline packaging/systemd/microshift-containerized.service:6]:
```suggestion
Description=MicroShift Containerized
```

This is what the final user will see, can we keep it ? I know podman will auto-generate the other

**Review by mangelajo — CHANGES_REQUESTED** [review]
just let's try to save the description.

I believe hostnet is the right way for microshift containerized! :)

**Comment by husky-parul** [comment]
> just let's try to save the description.
> 
> I believe hostnet is the right way for microshift containerized! :)

Done.

**Pull request merged**